lib/server/exashareresolver.py

```diff
@@ -32,19 +32,15 @@ def supports(url):
 
 
 def resolve(url):
-    data = re.search(r'<script[^\.]+?\.setup\((.+?)\);', util.request(url), re.I | re.S)
+    realurl = re.search(r'<iframe src="([^"]+)".*', util.request(url), re.I | re.S).group(1)
+    data = re.search(r'<script[^\.]+?\.setup\((.+?)\);', util.request(realurl), re.I | re.S)
     if data:
         data = data.group(1).decode('string_escape')
-        data = re.sub(r'\w+\(([^\)]+?)\)', r'\1', data)  # Strip JS functions
+        data = re.sub(r'\w+\(([^\)]+?)\)', r'\1', data).replace('primaryCookie','"primaryCookie"')  # Strip JS functions
```

Comment:
Do we really have to change this line ? Demjson should parse it even without this change.